src/tools/rbd_mirror/image_sync/SnapshotCopyRequest.h

```diff
@@ -104,6 +104,7 @@ class SnapshotCopyRequest : public BaseRequest {
   librados::snap_t m_prev_snap_id = CEPH_NOSNAP;
 
   std::string m_snap_name;
+  cls::rbd::SnapshotNamespace m_snap_namespace;
```

Comment:
Minor: should only handle UserSnapshotNamespace types -- skip GroupSnapshotNamespace